x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/permission/Role.java

```diff
@@ -228,6 +228,10 @@ public Builder add(
             return this;
         }
 
+        public Builder addRemoteGroup(final Set<String> remoteClusterAliases, final IndexPrivilege privilege, final String... indices) {
+            return addRemoteGroup(remoteClusterAliases, FieldPermissions.DEFAULT, null, privilege, false, indices);
+        }
```

Comment:
This method seems to be test only. Do we really need it?

Reply:
Yup, will nuke.